### ISSUE TITLE
Базовые операторы в JS

### ISSUE THREAD

**Issue by furtivite** [doka-guide/content#5077]
### Тема

Надо рассказать о самых базовых операторах в JavaScript: `+`, `-`, `*`, `/`, `%`, `**`. Добавить про инкремент и декремент, рассказать про приоритет и присваивание.

### Зачем

Так случилось, что такой материал пока что отсутствует в Доке. Можно исправить ситуацию, потому что базовыми операторами мы пользуемся каждый день

### Кто напишет материал

- [ ] Я сам напишу этот материал

**Comment by v-yan-v** [comment]
Про основные арифметические операции в Доке уже написано в статье про [числа](https://doka.guide/js/number/). Может стоит дополнить эту статью информацией про приоритет операций, инкремент/декремент и сокращённое присваивание (`+=` и т.п.) ?
А в статье про [строки](https://doka.guide/js/string/) так же рассказано про оператор `+`. 

Так что насколько могу видеть, не хватает отдельной статьи только на тему  `побитовые операции`

**Comment by anton-fomichev** [comment]
Хочу взять задачу

**Comment by vitya-ne** [comment]
Привет, отлично!
Закрепил за тобой

**Comment by anton-fomichev** [comment]
Тема довольно простая, поэтому есть вопрос: какой объем статьи здесь предпочтителен? Не хочется перегружать большим количеством тонкостей в такой теме, особенно новичков. Сейчас получилось ~10 мин (до редактуры). На мой взгляд, выглядит +- оптимально

**Comment by vitya-ne** [comment]
Ориентируйся на соседние доки )
Обычно контент ~ 5мин чтения.
В любом случае ревьюверы направят и подскажут

**Comment by furtivite** [comment]
Возможно, я тут зря врываюсь, но в тегах здень не дока, а обзорная статья. Как пример можно взять гайд по флексам. @HellSquirrel, можешь уточнить?

**Comment by vitya-ne** [comment]
@furtivite Егор, у меня вопросы по теме )

> Надо рассказать о самых базовых операторах в JavaScript: +, -, *, /, %, **. Добавить про инкремент и декремент

Перечисленные операторы на [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators#arithmetic_operators) называются арифметическими.

Если оставить в  заголовке "базовые", то как насчёт не менее базовых логических операторов ( '!', `&&`, `||`, `??`) ?

**Comment by anton-fomichev** [comment]
@vitya-ne 
Если в эту же доку вставим `!`, `&&`, `||`, `??`, то раздуем её и выйдем за рамки 10 минут. Я бы предложил `!`, `&&`, `||`, `??` вынести в отдельную доку. Разделённые доки помогут сохранить фокус на важных вещах и не перегрузят читателя

**Comment by vitya-ne** [comment]
> Если в эту же доку вставим !, &&, ||, ??, 

Да, и к тому же у нас уже есть дока про логические операторы. Это я просто насчёт названия статьи уточнял

**Comment by anton-fomichev** [comment]
@vitya-ne 
Как в итоге поступим? Смущает то, что если сужать название до "арифметических" операторов, то общая информация об операторах кажется неуместной

**Comment by vitya-ne** [comment]
Оставим как есть )
